src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
```

Comment:
I think that is an ngDocs thing ".content" not patternfly so we need to stop that from affecting the examples since it is messing with our styles. But I agree with @waldenraines we either need to use the h3 or a class name that is relevent to the content not the tag.